.github/workflows/build.yml

```diff
@@ -59,7 +29,7 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - name: Install dependencies
-        run: npm install --build-from-source
```

Comment:
This needs to stay, otherwise CI breaks. It also shouldn't ever try to fetch prebuilts in CI